src/ResourceManager/AzureStackStorage/Commands.AzureStackStorage/Validation/ValidateAbsoluteUri.cs

```diff
@@ -12,7 +12,6 @@
 // limitations under the License.
 // ----------------------------------------------------------------------------------
```

Comment:
I cannot see .csproj changes for the new SDK reference, please make sure that you are updating SDK version as well